src/backend/metal/src/command.rs

```diff
@@ -500,6 +500,15 @@ impl core::RawCommandBuffer<Backend> for CommandBuffer {
         unimplemented!()
     }
 
+    fn bind_graphics_descriptor_sets(
```

Comment:
typo

Reply:
ofc in the only backend I can't check locally :sweat_smile:
thanks for reviewing!

bors r=kvark